pypdf/_writer.py

```diff
@@ -1832,6 +1830,8 @@ def remove_images(self, ignore_byte_string_object: bool = False) -> None:
         )
         for page in pages:
             page_ref = cast(DictionaryObject, self.get_object(page))
+            if "/Contents" not in page_ref:
+                return
```

Comment:
ROB: This change might be surprising in this PR

Reply:
agree : the test reactivation below and this change would be better in another PR

Reply:
You're right :+1: 

I moved this to https://github.com/py-pdf/pypdf/pull/1598